### DISCUSSION TITLE
How to change the shortcut of snip?

### DISCUSSION THREAD

**Discussion by note286** [lukas-blecher/LaTeX-OCR#342] [Q&A]
I install on Windows 10 by

```
pip install "pix2tex[gui]"
```

and run with

```
pix2tex_gui
```

I want to change the shortcut of snip, I found the code is

https://github.com/lukas-blecher/LaTeX-OCR/blob/1781514fb8c92ea9f94057295fdae0e683f4648e/pix2tex/gui.py#L66

Is there any way to modify this shortcut key? Is there any parameter that can be passed into `pix2tex_gui` command to control?

**Comment by Sazzzzzz** [comment]
There are three aspects to adjust. Substitute `Alt+S` in lines 135, 138, and 179 with your preferred shortcut (I have set it to `Shift+Alt+S`; you may simply replace all instances of `Alt+S`), and it will function seamlessly!  

The modification might vary slightly if you are not operating on Windows.
![image](https://github.com/user-attachments/assets/fdd1a023-9ccc-40a4-8528-460fd745bdcb)
